### DISCUSSION TITLE
Adopt crowd translation service

### DISCUSSION THREAD

**Discussion by nevack** [losnoco/Cog#303] [Ideas]
For now Cog is translated into English and Spanish.
There's possibility that using some crowd translation service (like Crowdin, Transifex, etc.) will help us to translate Cog into more languages.

Translation services require to import and export strings.
This step can be automated, I think.
But it's just a way easier to translate something when you don't need to checkout repo, learn some new file format like Xcode's `.strings`.
You can just open Web UI and submit key translations, some services even provide automated translations based on other open source projects.

As for myself, I can help to create full Belarusian and Russian translations.

**Comment by paxcoder** [comment]
Hi.
I wrote Polish translation, I hope that the syntax is 100% correct, I am new to this translation system, and the web UI to translate stuff would be better, I would have less job figuring out what the heck I need to translate and what *not*, without reading c code.

**Comment by kode54** [comment]
We are using Lokalise internally, maybe we should invite you to edit your translation there? I'm not sure how that works for open source projects like this.

**Comment by paxcoder** [comment]
it should be great, I didn't tried this one jet, but if it is accessible with screenreader, it should be no problem, and probably make translation faster.

**Comment by kddlb** [comment]
I'm in talks with Lokalise for an open-source team license.

**Reply by kddlb** [reply]
The Free Open Source plan was activated for Cog!

**Reply by nevack** [reply]
Nice! How can we contribute now?

**Reply by kddlb** [reply]
I need an email address so I can add people as contributors.

**Reply by kddlb** [reply]
(Send to kevin@kddlb.cl)

**Comment by paxcoder** [comment]
ok, the main creator of this app has my mail.
